### PR TITLE
[14.0][MIG] stock_measuring_device_zippcube

### PR DESCRIPTION
**Pull request by hailangvn**
This PR depends on below PRs.
https://github.com/OCA/stock-logistics-warehouse/pull/1242
https://github.com/OCA/product-attribute/pull/908

### CODE REVIEW

**Review by sebalix — COMMENTED** [review]
LG, waiting for https://github.com/OCA/stock-logistics-warehouse/pull/1242 to be fixed

**Comment by simahawk** [comment]
LG but waiting for #1242 to merge

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo14-addon-stock_measuring_device @git+https://github.com/hailangvn/stock-logistics-warehouse@14.0-mig-stock_measuring_device#subdirectory=setup/stock_measuring_device
+odoo14-addon-product_packaging_dimension @git+https://github.com/hailangvn/product-attribute@14.0-fix-product_packaging_dimension-compute_volume#subdirectory=setup/product_packaging_dimension
```
Comment (mmequignon) [inline test-requirements.txt]: better reference the pull-requests instead :
```
odoo14-addon-stock_measuring_device @ git+https://github.com/OCA/stock-logistics-warehouse.git@refs/pull/<PR_ID>/head#subdirectory=setup/stock_measuring_device
```

**Review by mmequignon — APPROVED** [review]

File: test-requirements.txt (hunk 1 above)
Comment (simahawk) [reply test-requirements.txt]: @hailangvn can you drop it?

Comment (hailangvn) [reply test-requirements.txt]: Thanks @mmequignon, @simahawk. I have just updated the PR to drop test dependencies.

**Pull request merged**